### PR TITLE
Start agent pair from CI

### PR DESCRIPTION
**Pull request by mosheka**
1. Move from the Makefile to the CI
2. Move from reading the node id from the stdout to the file system

### CODE REVIEW

File: tests/base_test_agent.py [hunk 1]
```diff
@@ -60,16 +60,14 @@ def send(self, data):
...
-        pattern = re.compile(u'.*NodeID (\w+)')
         node_id = 'NULL'
-        for line in open('/opt/logs/node.log', "r", encoding="utf-8"):
-            results = pattern.match(line)
-            if results != None:
-                node_id = results.group(1)
-                break;
+        try:
+            node_id = next(os.walk('/opt/logs'))[1][0]
+        except:
```
Comment (gavraz) [inline tests/base_test_agent.py]: Maybe except the specific exception(s) you can get here?

File: tests/base_test_agent.py [hunk 2]
```diff
@@ -60,16 +60,14 @@ def send(self, data):
...
+            print('Error finding log folder')
+        print('NodeId:' + node_id)
         return node_id
```
Comment (gavraz) [inline tests/base_test_agent.py]: Shouldn't you terminate?
Is node_id now evaluated to zero? what is the meaning of this value?

Comment (mosheka) [reply tests/base_test_agent.py]: I assume it will remain NULL and test will fail

File: tests/base_test_agent.py (hunk 1 above)
Comment (mosheka) [reply tests/base_test_agent.py]: Done

**Pull request merged**